components/thumbnail/basic/src/index.js

```diff
@@ -1,15 +1,27 @@
 import React, {Component} from 'react'
 import PropTypes from 'prop-types'
+import ImagePlaceholder from '@schibstedspain/sui-image-placeholder'
 
 class ThumbnailBasic extends Component {
+  static BASE_CLASS = 'sui-ThumbnailBasic'
+  static IMAGE_CLASS = 'sui-ThumbnailBasic-image'
+  static CAPTION_CLASS = 'sui-ThumbnailBasic-caption'
+
   render () {
-    const { image, captionText } = this.props
+    const {captionText} = this.props
     return (
-      <figure className='sui-ThumbnailBasic'>
-        <img className='sui-ThumbnailBasic-image' src={image.src} alt={image.alt} />
-        <figcaption className='sui-ThumbnailBasic-caption'>
-          <span>{captionText}</span>
-        </figcaption>
+      <figure className={ThumbnailBasic.BASE_CLASS}>
+        <ImagePlaceholder
+          image={this.props.image}
+          placeholder={this.props.placeholder}
+          fallbackImage={this.props.fallbackImage}
+        />
+        {
+          captionText &&
+            <figcaption className={ThumbnailBasic.CAPTION_CLASS}>
+              <span>{captionText}</span>
+            </figcaption>
```

Comment:
This span is necesary?

Reply:
I changed it to a div and added a class for it (it was using the tag as selector)